### PR TITLE
Regular updates (Apr 16)

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- Address https://github.com/iterative/dvc.org/issues/1123#issuecomment-614929585
- Complete #1097 by removing mentions of `raw` metrics type
- Add --summary option to checkout cmd ref.

### CODE REVIEW

File: content/docs/command-reference/metrics/add.md [hunk 1]
```diff
@@ -106,7 +104,7 @@ outs:
     path: metrics.json
     cache: true
     metric:
-      type: raw
+      type: json
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/add.md:107]: This change is correct right @dmpetrov? See https://dvc.org/doc/command-reference/metrics/add#examples

I also removed all mentions of `raw` type metrics in this PR.

File: content/docs/command-reference/checkout.md [hunk 2]
```diff
@@ -69,16 +69,18 @@ be pulled from remote storage using `dvc pull`.
 
 ## Options
 
+- `--summary` - show summary of the changes.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/checkout.md:72]: I noticed checkout now has this option so I'm adding it. But this description isn't great, where can I find more details? I'm guessing @efiop may know. I.e. summary of what changes? Thanks!

Comment (shcheklein) [reply content/docs/command-reference/checkout.md:72]: --summary is covered in the Saugat's PR that we have been trying to merge for while

Comment (jorgeorpinel) [reply content/docs/command-reference/checkout.md:72]: Ah yes I see there's a note about it in https://github.com/iterative/dvc.org/pull/1054#pullrequestreview-391199946. OK.

But in this PR I also changed the order of the options so I say let's keep it here for now and we'll have to solve a little conflict later.

**Comment by shcheklein** [comment]
@jorgeorpinel merging this?

**Comment by jorgeorpinel** [comment]
Yes, thanks.

**Pull request merged**